migration/src/main/resources/migrations/ensure-order-type-has-java_class_name-column.xml

```diff
@@ -5,49 +5,121 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
 
-    <!-- In 1.10+, All users that created orders(orders.orderer) have provider accounts -->
 
+    <!-- Ensure order_type table has java_class_name and parent columns.  Credit k-joseph -->
     <changeSet id="ensure-order-type-has-java_class_name-column" author="IMB">
+        <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="order_type" columnName="java_class_name"/></not>
+            <not><columnExists tableName="order_type" columnName="parent"/></not>
+        </preConditions>
+        <comment>
+            Add java_class_name and parent columns to order_type table
+        </comment>
+        <addColumn tableName="order_type">
+            <column name="java_class_name" type="varchar(255)" />
+        </addColumn>
+        <addColumn tableName="order_type">
+            <column name="parent" type="int(11)" />
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="ensure-drug-order-type-has-java_class_name-value" author="IMB">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) FROM order_type
+                WHERE name = 'Drug Order' AND java_class_name IS NULL
+            </sqlCheck>
+        </preConditions>
+        <comment>Setting java_class_name column for Drug Order type row</comment>
+        <update tableName="order_type">
+            <column name="java_class_name" value="org.openmrs.DrugOrder" />
+            <where>name='Drug order'</where>
+        </update>
+    </changeSet>
+
+    <changeSet id="ensure-drug-order-type-has-java_class_name-value-2" author="IMB">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) FROM order_type
+                WHERE name = 'Drug' AND java_class_name IS NULL
+            </sqlCheck>
+        </preConditions>
+        <comment>Setting java_class_name column for Drug type row(Butaro)</comment>
+        <update tableName="order_type">
+            <column name="java_class_name" value="org.openmrs.DrugOrder" />
+            <where>name='Drug'</where>
+        </update>
+    </changeSet>
```

Comment:
Might be worth considering cleaning all of these duplicate order types up at this time.  Probably a different ticket to do so, but this is a good opportunity.